sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/AlertQueryTimeMode.cs

```diff
@@ -8,7 +8,7 @@ namespace Azure.AI.MetricsAdvisor
 {
     /// <summary>
     /// Specifies which time property of an <see cref="AnomalyAlert"/> will be used to filter results
-    /// in the <see cref="MetricsAdvisorClient.GetAlerts"/> and the <see cref="MetricsAdvisorClient.GetAlertsAsync"/>
+    /// in the "MetricsAdvisorClient.GetAlerts" and the "MetricsAdvisorClient.GetAlertsAsync"/>
```

Comment:
Why was `<see cref=` removed?  Is this because the method is no longer present?

Reply:
Yes